ios/MaasNearbyTransit/Sources/MaasNearbyTransit/Components/NearbyTransitFilterView.swift

```diff
@@ -0,0 +1,78 @@
+import SwiftUI
+import Swappable
+
+struct NearbyTransitFilterView: View {
+
+    public let items: [FilterItem]
+    public var didTapItem: (FilterItem.ID) -> ()
+    public var disabledItems: Set<FilterItem.ID>
+
+    var body: some View {
+        MultiSelectFilter(
+            items: items,
+            itemView: NearbyTransitFilterItemView.init,
+            didTapItem: didTapItem,
+            disabledItems: disabledItems
+        )
+    }
+}
+
+// MARK: - Preview
+#if DEBUG
+public struct NearbyTransitFilterView_Previews: PreviewProvider, Snapped {
```

Comment:
It's think we need feature components previews and snapshots, even if this is a wrapped up in another component with no constants, because we use a items inside that component that uses global constant and i think we would like to monitor changes in flows via snapshots if something changes.